src/lazy/binary/raw/v1_1/immutable_buffer.rs

```diff
@@ -51,7 +51,7 @@ pub struct BinaryBuffer<'a> {
 
 impl Debug for BinaryBuffer<'_> {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
-        write!(f, "BinaryBuffer {{")?;
+        write!(f, "BinaryBuffer @ offset={} {{", self.offset)?;
```

Comment:
🪧 Having the byte offset makes it much easier to debug things with `inspect` and/or a hex dump.